zzzz_modular_occulus/code/game/gamemodes/events/weather/rad_storm.dm

```diff
@@ -0,0 +1,7 @@
+////Radiation storms occur when the station passes through an irradiated area, and irradiate anyone not standing in protected areas (maintenance, emergency storage, etc.)
+///This is the rad_storm modular file for Occulus. Please add any additions to the radiation storm event here. Thanks!
+
+/datum/weather/rad_storm/New()
```

Comment:
This is not the correct way to do a modular override.

Please just re-define protected_areas. Do not overwrite the new() proc

Reply:
Fixed!